docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -156,6 +156,10 @@ Users can install and run multiple versions of the .NET Framework on their compu
 
 - The following example checks the value of the `Release` keyword to determine whether .NET Framework 4.6.2 or higher is installed (returning `True` if it is and `False` otherwise). You can substitute another value listed in the table to check for a different minimum-required .NET Framework version.
 
+```PowerShell
```

Comment:
I'd move the last sentence from the previous paragraph to after the code to introduce the table